resources/themes/cura-light/images/first_run_Ultimaker_cloud.svg

```diff
@@ -0,0 +1,16 @@
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <title>Group-cloud</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Iteration-4" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="9-Account-and-finish" transform="translate(-540.000000, -255.000000)">
+            <g id="Group-cloud" transform="translate(540.000000, 255.000000)">
+                <path d="M184.645934,101.269713 C184.402632,95.7100464 182.987439,90.4490704 180.637153,85.7196609 L184.851695,85.7196609 C187.817797,85.7196609 190.254237,83.3353195 190.254237,80.4326431 L190.36017,80.4326431 L190.36017,16.262763 C190.36017,15.1224259 189.40678,14.2930898 188.347457,14.2930898 L149.933791,14.2930898 L149.933791,63.9459005 C148.198887,63.7202321 146.428947,63.6037711 144.631348,63.6037711 C142.750807,63.6037711 140.900537,63.7312272 139.088982,63.9778345 L139.088982,14.2930898 L138.930084,14.2930898 L138.930084,11.4224041 C138.588831,10.5826557 137.742066,10.0297833 136.824681,10.0297833 L63.0561437,10.0297833 C61.7452327,10.0297833 60.7918428,11.0794119 60.7918428,12.2456657 L60.7918428,84.4367811 C60.7918428,87.7022917 63.5328385,90.3846758 66.8697029,90.3846758 L106.674877,90.3846758 C105.315869,94.3430337 104.579138,98.5831707 104.579138,102.99253 C104.579138,104.646987 104.68286,106.27762 104.884257,107.878484 L65.082097,107.878484 C63.771186,107.995109 62.460275,108.461611 61.5068852,109.277988 L60.7918428,109.977741 C60.0768004,111.143995 58.6467157,111.377246 57.216631,111.377246 L50.7812497,111.377246 C50.4237285,111.377246 50.0662073,111.027369 50.0662073,110.677493 L50.0662073,101.269713 L13.3474575,101.269713 C12.1822033,101.373379 11.0169491,101.788048 10.1694914,102.513717 L9.53389824,103.135719 C8.89830503,104.172389 7.62711862,104.379723 6.35593214,104.379723 L0.635593219,104.379723 C0.317796608,104.379723 -6.39488462e-14,104.068722 -6.39488462e-14,103.757721 L-6.39488462e-14,5.99972886 C-6.39488462e-14,5.68872783 0.317796608,5.3777268 0.635593219,5.3777268 L50.0662073,5.3777268 L50.0662073,0.699752327 C50.0662073,0.349876165 50.4237285,-1.63424829e-13 50.7812497,-1.63424829e-13 L149.218749,-1.63424829e-13 C149.57627,-1.63424829e-13 149.933791,0.349876165 149.933791,0.699752327 L149.933791,5.3777268 L199.364407,5.3777268 C199.682204,5.3777268 200,5.68872783 200,5.99972886 L200,103.757721 C200,104.068722 199.682204,104.379723 199.364407,104.379723 L193.644068,104.379723 C192.478814,104.276056 191.313559,103.861388 190.466102,103.135719 L189.830509,102.513717 C189.194915,101.477046 187.923729,101.269713 186.652543,101.269713 L184.645934,101.269713 Z M50.0662073,14.2930898 L11.5466101,14.2930898 C10.3813559,14.2930898 9.53389824,15.2260929 9.53389824,16.262763 L9.53389824,80.4326431 C9.53389824,83.3353195 11.9703389,85.7196609 14.9364406,85.7196609 L50.0662073,85.7196609 L50.0662073,14.2930898 Z" id="Combined-Shape" fill="#08073F" fill-rule="nonzero"></path>
```

Comment:
In the middle there are lots of entries to draw lines of 0.000000000000001px length. Those can be removed.

Reply:
I'm going to leave the SVGs as they are for now.

Reply:
Removed in 705cdedf0